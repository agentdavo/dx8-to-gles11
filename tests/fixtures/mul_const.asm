def c0, 1.0, 0.0, 0.0, 1.0
mul r0, v0, c0
