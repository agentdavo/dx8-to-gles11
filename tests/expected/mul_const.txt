LOAD_CONSTANT idx=0 1.0 0.0 0.0 1.0
TEX_ENV_COMBINE func=8448
